### PR TITLE
Proxy 7

### PR DESCRIPTION
**Pull request by kabiakou**
- replaced http js libarary with axios one
- added endpoint
- structured project
- added routes
- connected express js framework

### CODE REVIEW

File: js/mappers/MeteorMapper.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const buildMeteorEntity = (meteorStatistic) => {
+    
+    closeApprochDates = []
```
Comment (plutsenka) [inline js/mappers/MeteorMapper.js]: approAch?

File: js/routes/MeteorRoute.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const express = require('express')
+const router = express.Router()
+const meteorService = require('../services/MeteorService.js')
+
+
+router.get('/', async (req, res) => {
+    try {
+        res.json(await meteorService.getMeteorsData())
+    } catch (error) {
+        res.status(500).send(error)
```
Comment (plutsenka) [inline js/routes/MeteorRoute.js]: Would be better to inspect error - Nasa API can potentially return some error, e.g. too many requests, and 500 doesn't fit in this case. Generally it's bad idea to return 500's error to client, if server can process it

File: js/routes/MeteorRoute.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    }
+})
+
```
Comment (plutsenka) [inline js/routes/MeteorRoute.js:12]: extra empty line

File: js/services/MeteorService.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+const dateUtils = require('../utils/DateUtils.js')
+const requestUtils = require('../clients/MeteorClient.js')
+const meteorMapper = require('../mappers/MeteorMapper.js')
+
+const monday = dateUtils.getDayOfTheCurrentWeek(1)
```
Comment (plutsenka) [inline js/services/MeteorService.js]: What does 1 & 5 mean? It not obvious, so please avoid using anonymous values, try to add sense to them (e.g. mondayOffset or smth else)

File: js/services/MeteorService.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
+const dateUtils = require('../utils/DateUtils.js')
```
Comment (plutsenka) [inline js/services/MeteorService.js]: You don't need to use .js in require section

File: js/utils/DateUtils.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+const moment = require('moment')
```
Comment (plutsenka) [inline js/utils/DateUtils.js]: As was discussed in our chat, moment is not good library to use (I was  affected as well and changed it to date-fns)

File: js/routes/MeteorRoute.js (hunk 3 above)
Comment (kabiakou) [reply js/routes/MeteorRoute.js:12]: Fixed

File: js/services/MeteorService.js (hunk 4 above)
Comment (kabiakou) [reply js/services/MeteorService.js]: Thanks!
Replaced with date-fns

File: js/services/MeteorService.js (hunk 5 above)
Comment (kabiakou) [reply js/services/MeteorService.js]: Fixed

File: js/utils/DateUtils.js (hunk 6 above)
Comment (kabiakou) [reply js/utils/DateUtils.js]: Thanks!
Replaced with date-fns

File: js/mappers/MeteorMapper.js (hunk 1 above)
Comment (kabiakou) [reply js/mappers/MeteorMapper.js]: Fixed

File: js/routes/MeteorRoute.js (hunk 2 above)
Comment (kabiakou) [reply js/routes/MeteorRoute.js]: Thanks! Have added ExceptionHandler

**Pull request merged**